src/stylesheets/vendor/slick-dots.scss

```diff
@@ -0,0 +1,17 @@
+.slick-dots {
+  bottom: -55px;
+  /* stylelint-disable-next-line */
+  li.slick-active button::before {
+    color: map-get($brand-white, 'primary');
+  }
+
+  li button::before {
+    color: map-get($brand-white, 'primary');
+
+    opacity: 20%;
+  }
+
+  li button {
```

Comment:
@rafayet-monon Can we assign some classes for these elements?

Reply:
Unfortunately, it's in slick package. So, no way to assign it for now.

Reply:
😥